### PR TITLE
Subresource Integrity

### PR DESCRIPTION
**Pull request by Natim**
https://hacks.mozilla.org/2015/09/subresource-integrity-in-firefox-43/

We can use this for Kinto.js releases !!

### CODE REVIEW

**Comment by michielbdejong** [comment]
Nice! \o/

**Comment by Natim** [comment]
Rebased. r? @n1k0

**Comment by ametaireau** [comment]
Looks good to me. Maybe add a section stating that you need openssl in order to run the commands?

File: docs/installation.md [hunk 1]
```diff
@@ -25,3 +25,24 @@ Dev and production ready assets are available in the [`dist` folder of the repos
 
 - Dev version, including source maps: [kinto.dev.js](https://cdn.rawgit.com/Kinto/kinto.js/1.0.0-rc.1/dist/kinto.dev.js)
 - Production version, minified, no source maps: [kinto.dev.js](https://cdn.rawgit.com/Kinto/kinto.js/1.0.0-rc.1/dist/kinto.min.js)
+
+
+### Subresource integrity
+
+To make sure that you are using the right code when loading from a CDN, you can use subresource
+integrity with the hash provided in ``kinto.min.sha384`` with your release.
```
Comment (ametaireau) [inline docs/installation.md]: We could add something like "You can see the compatible browsers at http://caniuse.com/#search=SRI". I'm not sure why it doesn't show up there. @mozfreddyb any idea?

Comment (mozfreddyb) [inline docs/installation.md]: Soon. https://github.com/Fyrd/caniuse/issues/1170

**Comment by mozfreddyb** [comment]
> Looks good to me. Maybe add a section stating that you need openssl in order to run the commands?


You could also use something nodejs based if that fits better to your build environment. See https://github.com/neftaly/npm-sri-toolbox and https://github.com/odino/node-sri/

File: docs/installation.md (hunk 1 above)
Comment (Natim) [inline docs/installation.md]: It now show there thanks :)

File: docs/installation.md [hunk 2]
```diff
@@ -25,3 +25,24 @@ Dev and production ready assets are available in the [`dist` folder of the repos
...
+
+.. code-block:: html
+
+    <script src="https://cdn.rawgit.com/Kinto/kinto.js/1.0.0-rc.1/dist/kinto.min.js"
+            integrity="sha384-2EBP0zFAKrL1idZDEzowmRmkeEFAneGUwGUHKmGSWKtTtHRQPbyOJyCk0xCgT1pC"
+            crossorigin="anonymous">
+    </script>
+
+Where ``2EBP0zFAKrL1idZDEzowmRmkeEFAneGUwGUHKmGSWKtTtHRQPbyOJyCk0xCgT1pC``
+is the value in ``dist/kinto.min.sha384``.
```
Comment (leplatrem) [inline docs/installation.md]: Where do I find this value? Online in gh-pages branch?

File: docs/installation.md [hunk 3]
```diff
@@ -25,3 +25,24 @@ Dev and production ready assets are available in the [`dist` folder of the repos
...
+
+If you need to calculate your subresource integrity hash please use
+https://srihash.org/.
```
Comment (leplatrem) [inline docs/installation.md]: When would I need to calculate it?

File: docs/installation.md (hunk 2 above)
Comment (Natim) [inline docs/installation.md]: You can calculate it yourself and yes with this PR you can also get the last version on the github repository along with the correct file.

**Comment by leplatrem** [comment]
:+1: r+

File: docs/installation.md [hunk 4]
```diff
@@ -16,10 +16,29 @@ $ npm install kinto --save
...
 
 > #### Notes
 >
 > Only stable tags are released as dist files; if you plan on using latest versions from master, you need to [build them manually](hacking.md#generating-dist-files).
+
+
+### Subresource integrity
+
+To make sure that you are using the right code when loading from a CDN, you can use subresource
+integrity with the hash provided below:
```
Comment (leplatrem) [inline docs/installation.md]: nit: add a link to https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity ?

**Comment by n1k0** [comment]
One thing, can't we have a command for updating the index.html file with the right values?

Updating it manually everytime we push a new release is not undoable, but would be nice to smooth the already quite complicated release procedure a little.

**Comment by n1k0** [comment]
My bad, we're only doing this for bootstrap. r+ :+1:

**Pull request merged**